java/arcs/core/data/testdata/BUILD

```diff
@@ -46,3 +48,15 @@ arcs_kt_gen(
     ],
     data = [":shared_schemas"],
 )
+
+arcs_proto_plan(
+    name = "example-proto",
+    src = "WriterReaderExample.arcs",
+)
+
+arcs_kt_plan_2(
+    name = "plan_from_proto",
+    srcs = [":example-proto"],
+    package = "arcs.core.data.testdata",
+    visibility = ["//visibility:public"],
```

Comment:
Is the default visibility in this package `//java/arcs:allowed-packages` not good?

Reply:
It is in this PR, which is a branch-off from another where I implement code-generation. There, I use the output of this rule in a goldens test.

Reply:
I can rm the visibility argument here, and we can evaluate its use in a future PR.

Reply:
Sounds good to me.